packages/moonstone/VideoPlayer/MediaSliderDecorator.js

```diff
@@ -174,6 +178,11 @@ const MediaSliderDecorator = hoc((config, Wrapped) => {	// eslint-disable-line n
 			this.move(ev.clientX);
 		}
 
+		handleTouchMove (ev) {
+			// ignores multi touch
+			this.move(ev.touches[0].clientX);
```

Comment:
Should we round these numbers?  As opposed to mouse movements these appear to contain floats.

Reply:
I'm not sure if it would be useful to round numbers. The proportion calculation will end up with floating points most of the time even with the rounded numbers. (https://github.com/enactjs/enact/blob/46677e3b64a25bac680e62227d9d7065d85db523/packages/moonstone/VideoPlayer/MediaSliderDecorator.js#L135)

Reply:
I figured you might say that.  I suspect we won't see a lot of switching between touch and mouse in the same session so it's not really a big issue for optimization.